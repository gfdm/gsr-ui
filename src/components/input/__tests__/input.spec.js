import { mount, createLocalVue } from '@vue/test-utils'
import IdInput from '../'

const localVue = createLocalVue()
localVue.use(IdInput)

describe('IdInput', () => {
  test('should render well', () => {
    const Wrapper = {
      name: 'wrap',
      template: `
        <id-input type="text" placeholder="Please input">
        </id-input>
      `
    }

    const wrapper = mount(Wrapper, {
      localVue
    })

    expect(wrapper.html()).toMatchSnapshot()
  })
})
