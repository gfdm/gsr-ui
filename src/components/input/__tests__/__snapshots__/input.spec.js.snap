// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IdInput should render well 1`] = `
<section class="id-input" placeholder="Please input">
  <input placeholder="Please input">
</section>
`;
